camel-core/src/main/java/org/apache/camel/ComponentVerifier.java

```diff
@@ -27,71 +27,91 @@
  * Defines the interface used to validate component/endpoint parameters.
  */
 public interface ComponentVerifier {
-    String CODE_AUTHENTICATION = "authentication";
-    String CODE_EXCEPTION = "exception";
-    String CODE_INTERNAL = "internal";
-    String CODE_MISSING_OPTION = "missing-option";
-    String CODE_MISSING_OPTION_GROUP = "missing-option-group";
-    String CODE_UNKNOWN_OPTION = "unknown-option";
-    String CODE_ILLEGAL_OPTION = "illegal-option";
-    String CODE_ILLEGAL_OPTION_GROUP_COMBINATION = "illegal-option-group-combination";
-    String CODE_ILLEGAL_OPTION_VALUE = "illegal-option-value";
-    String CODE_INCOMPLETE_OPTION_GROUP = "incomplete-option-group";
-    String CODE_UNSUPPORTED = "unsupported";
-    String CODE_UNSUPPORTED_SCOPE = "unsupported-scope";
-    String ERROR_TYPE_ATTRIBUTE = "error.type";
+
     String ERROR_TYPE_EXCEPTION = "exception";
     String ERROR_TYPE_HTTP = "http";
-    String HTTP_CODE = "http.code";
-    String HTTP_TEXT = "http.text";
-    String HTTP_REDIRECT = "http.redirect";
-    String HTTP_REDIRECT_LOCATION = "http.redirect.location";
-    String EXCEPTION_CLASS = "exception.class";
-    String EXCEPTION_INSTANCE = "exception.instance";
-    String GROUP_NAME = "group.name";
-    String GROUP_OPTIONS = "group.options";
 
     enum Scope {
-        NONE,
         PARAMETERS,
         CONNECTIVITY;
 
-        private static final Scope[] VALUES = values();
-
-        public static Scope fromString(String scope) {
-            for (int i = 0; i < VALUES.length; i++) {
-                if (ObjectHelper.equal(scope, VALUES[i].name(), true)) {
-                    return VALUES[i];
+        public static Scope fromString(String scopeS) {
```

Comment:
I will revert the whole method back. Just wondered about the private static `VALUES`